Entities/ProductEntity.java

```diff
@@ -0,0 +1,67 @@
+package Entities;
+
+import java.util.List;
+
+public class ProductEntity {
+    //Attributes
+    private int productId;
```

Comment:
Please avoid the use of primitive data types.

Reply:
Also Create a `BaseEntity` with the attributes that will be common between entities with attributes as `entityStatus`, `createdBy`, `updatedBy`, `createdDate`, `updatedDate`. The others entities will extend from this one.

Reply:
Changes from primitive to non-primitive have been done.

But, creating another entity will be against the stament number 4 from the project requeriments, so we can do it in another PR.

Reply:
Could you create another PR with the base entity first?